examples/simple-service-poc/simple_service.py

```diff
@@ -59,6 +59,8 @@ async def get_payload():
 
     async def start(self):
         # handler responsible for starting the service
+        self._ctx.deploy()
+        self._ctx.start()
```

Comment:
I'd rather be in favor of:

```suggestion
        async for script in super().start():
            yield script
```